src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/RazorCodeActionContext.cs

```diff
@@ -9,31 +9,20 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions;
 
-internal sealed class RazorCodeActionContext
+internal sealed class RazorCodeActionContext(
```

Comment:
record class?

Reply:
I definitely agree with that. Keep it sealed though.